CHANGES

```diff
@@ -0,0 +1,3 @@
+Next release
```

Comment:
@dkent Noticed one thing. you should not add this line to this file, you should add it to docs/release-notes.rst instead. Please move it there before i can merge this.